AWS/DirectoryInsights/get-jcdirectoryinsights.py

```diff
@@ -46,6 +48,13 @@ def jc_directoryinsights(event, context):
             raise Exception(f"Error - Service List contains 'all' and additional services : {serviceList}")
     finalData = []
 
+    if len(serviceList) > 1:
```

Comment:
I'm unsure about this, If the two strings generated are the same why would we break this out into a if statement?

Reply:
If there are >1 services, it creates multiple log for each service

Reply:
We reviewed and will produce multiple logs for multiple services